doc-content/enterprise-only/openshift/architecture-authoring-con.adoc

```diff
@@ -0,0 +1,92 @@
+
+When you use the {CENTRAL} user interface to deploy or manage a service on the {KIE_SERVER}, the {KIE_SERVER} receives the request through the Controller connection WebSocket. To deploy a service, the {KIE_SERVER} can request the necessary artifact from the Maven repository that is a part of {CENTRAL}. 
+
+Client applications use a REST API to use services that run on the {KIE_SERVER}.
+
+.Architecture diagram for a single authoring environment
+image::Overview/architecture-authoring-nonha.png[]
+
+[float]
+== Clustering {KIE_SERVERS} and using multiple {KIE_SERVERS}
+
+You can scale a {KIE_SERVER} pod to run a clustered {KIE_SERVER} environment.
+ifdef::PAM[]
+To scale a {KIE_SERVER}, you must ensure that it uses a database server in a separate pod or an external database server, and not a built-in H2 database engine.
+endif::PAM[]
+
+In a clustered deployment, several instances of the {KIE_SERVER} run the same services. These servers can connect to the {CENTRAL} Controller using the same server ID, so they can receive the same requests from the controller. {OPENSHIFT} provides load-balancing between the servers. 
+ifdef::PAM[]
+Decision services and business optimizer services that run on a clustered {KIE_SERVER} 
+endif::PAM[]
+ifdef::DM[]
+The services that run on a clustered {KIE_SERVER}
+endif::DM[]
+must be stateless, because requests from the same client might be processed by different instances.
+
+You can also deploy several independent {KIE_SERVERS} to run different services. In this case, the servers connect to the {CENTRAL} Controller with different server ID values. You can use the {CENTRAL} UI to deploy services to each of the servers.
+
+[float]
+== Smart Router
+
+The optional Smart Router component provides a layer between client applications and {KIE_SERVERS}. It can be useful if you are using several independent {KIE_SERVERS}.
+
+The client application can use services running on different {KIE_SERVERS}, but always connects to the Smart Router. The Smart Router automatically passes the request to the {KIE_SERVERS} that runs the required service. The Smart Router also enables management of service versions and provides an additional load-balancing layer.
+
+[float]
+== High-availability authoring environment
+
+In a high-availability (HA) authoring environment, the {CENTRAL} pod is scaled, so several instances of {CENTRAL} are running. {OPENSHIFT} provides load balancing for user requests. This environment provides optimal performance for multiple users and supports failover.
+
+Each instance of {CENTRAL} includes the Maven repository for the built artifacts and uses the ``.niogit` Git repository for source code. The instances use shared persistent storage for the repositories. A persistent volume with ReadWriteMany access is required for thsi storage.
```

Comment:
Is it ReadWrite or ReadWriteMany? 

suggested rewrite:

... access is required for this storage. (typo)